src/DotLiquid.Tests/TemplateTests.cs

```diff
@@ -8,47 +8,56 @@ namespace DotLiquid.Tests
         {
-            CollectionAssert.AreEqual(new[] { " " }, Template.Tokenize(" "));
-            CollectionAssert.AreEqual(new[] { "hello world" }, Template.Tokenize("hello world"));
+            CollectionAssert.AreEqual(new[] { " " }, Tokenize(" "));
+            CollectionAssert.AreEqual(new[] { "hello world" }, Tokenize("hello world"));
         }
 
         [Test]
         public void TestTokenizeVariables()
         {
-            CollectionAssert.AreEqual(new[] { "{{funk}}" }, Template.Tokenize("{{funk}}"));
-            CollectionAssert.AreEqual(new[] { " ", "{{funk}}", " " }, Template.Tokenize(" {{funk}} "));
-            CollectionAssert.AreEqual(new[] { " ", "{{funk}}", " ", "{{so}}", " ", "{{brother}}", " " }, Template.Tokenize(" {{funk}} {{so}} {{brother}} "));
-            CollectionAssert.AreEqual(new[] { " ", "{{  funk  }}", " " }, Template.Tokenize(" {{  funk  }} "));
+            CollectionAssert.AreEqual(new[] { "{{funk}}" }, Tokenize("{{funk}}"));
+            CollectionAssert.AreEqual(new[] { " ", "{{funk}}", " " }, Tokenize(" {{funk}} "));
+            CollectionAssert.AreEqual(new[] { " ", "{{funk}}", " ", "{{so}}", " ", "{{brother}}", " " }, Tokenize(" {{funk}} {{so}} {{brother}} "));
+            CollectionAssert.AreEqual(new[] { " ", "{{  funk  }}", " " }, Tokenize(" {{  funk  }} "));
         }
 
         [Test]
         public void TestTokenizeBlocks()
         {
-            CollectionAssert.AreEqual(new[] { "{%assign%}" }, Template.Tokenize("{%assign%}"));
-            CollectionAssert.AreEqual(new[] { " ", "{%assign%}", " " }, Template.Tokenize(" {%assign%} "));
+            CollectionAssert.AreEqual(new[] { "{%assign%}" }, Tokenize("{%assign%}"));
+            CollectionAssert.AreEqual(new[] { " ", "{%assign%}", " " }, Tokenize(" {%assign%} "));
 
-            CollectionAssert.AreEqual(new[] { " ", "{%comment%}", " ", "{%endcomment%}", " " }, Template.Tokenize(" {%comment%} {%endcomment%} "));
-            CollectionAssert.AreEqual(new[] { "  ", "{% comment %}", " ", "{% endcomment %}", " " }, Template.Tokenize("  {% comment %} {% endcomment %} "));
+            CollectionAssert.AreEqual(new[] { " ", "{%comment%}", " ", "{%endcomment%}", " " }, Tokenize(" {%comment%} {%endcomment%} "));
+            CollectionAssert.AreEqual(new[] { "  ", "{% comment %}", " ", "{% endcomment %}", " " }, Tokenize("  {% comment %} {% endcomment %} "));
         }
 
         [Test]
         public void TestInstanceAssignsPersistOnSameTemplateObjectBetweenParses()
         {
             Template t = new Template();
-            Assert.AreEqual("from instance assigns", t.ParseInternal("{% assign foo = 'from instance assigns' %}{{ foo }}").Render());
-            Assert.AreEqual("from instance assigns", t.ParseInternal("{{ foo }}").Render());
+            Assert.AreEqual("from instance assigns", t.ParseInternal("{% assign foo = 'from instance assigns' %}{{ foo }}", SyntaxCompatibility.DotLiquid22).Render());

```

Comment:
2.2 would be the new default syntax compatibility version, so only explicit 2.1 version compatibility input is required, is it not?

Reply:
ParseInternal has no default - it is internal. The default, however, is actually 2.0 to introduce no breaking changes. This is computed in Parse method based on Template.DefaultSyntax property.
The reason why I specified 2.2 is because the only difference in handling is whitespace and there is no whitespace in this. By specifying 2.2 I ensure no regression with this commit. It could go either way.